pyproject.toml

```
@@ -28,7 +28,7 @@ keywords = ["taskiq", "tasks", "distributed", "async"]
 [tool.poetry.dependencies]
 python = "^3.8.1"
 typing-extensions = ">=3.10.0.0"
-pydantic = "^1.6.2"
+pydantic = "<=2.0"
```

It's a bit incorrect. I would suggest to set it as:

```suggestion
pydantic = ">=1.0,<=3.0"
```